### ISSUE TITLE
Diff between bill readings

### ISSUE THREAD

**Issue by heinermann** [michaelmulley/openparliament#69]
Sometimes there are minor changes between a bill's readings that can be difficult to notice and it is not clear what has changed between readings.

So I'm requesting a feature to display the differences between readings on a bill to highlight what changed, similar to how we see code diffs on Github.

**Comment by JohnTendik** [comment]
@heinermann like this? https://myparliament.ca/bills/45-1/C-5?pageTab1=text&compareMode=true&v1=5792&v2=5795

**Comment by heinermann** [comment]
Yeah exactly that, looks good. Do you know how to contact the developer for myparliament? I've already run into issues I'd like to report.
